docs/integration/camel-dhis2-component.md

```diff
@@ -0,0 +1,308 @@
+
+`unmarshal().json(org.hisp.dhis.api.model.v40_0.WebMessage.class)`
+
+### delete
+
+Specify `delete` for the API name and `resource` for the method name to delete a DHIS2 resource as demonstrated in this route:
+
+```java
+from("direct:deleteResource")
+    .to("dhis2://delete/resource?path=organisationUnits/jUb8gELQApl&username=admin&password=district&baseApiUrl=https://play.dhis2.org/40.2.1/api")
+    .unmarshal().json(org.hisp.dhis.api.model.v40_0.WebMessage.class)
+    .choice()
+    .when(exchange -> !exchange.getMessage().getBody(org.hisp.dhis.api.model.v40_0.WebMessage.class).getStatus().equals(org.hisp.dhis.api.model.v40_0.WebMessage.Status.OK))
+        .log(LoggingLevel.ERROR, "Import error from DHIS2 while deleting org unit => ${body}")
+    .end();
+```
+
+The above route deletes the organisation unit `jUb8gELQApl` in DHIS2 thanks to the `dhis2://delete/resource` endpoint:
+
+`to("dhis2://delete/resource?path=organisationUnits/jUb8gELQApl&username=admin&password=district&baseApiUrl=https://play.dhis2.org/40.2.1/api")`
+
+`organisationUnits/jUb8gELQApl` is the `path` parameter which identifies the path where the resource to be deleted is located in DHIS2. A resource can be added to the HTTP request sent by setting the message body to the resource. The body produced from `dhis2://put/resource` is of type `java.io.InputStream`. In this example, the route deserialises the reply into a `WebMessage` object with:
+
+`unmarshal().json(org.hisp.dhis.api.model.v40_0.WebMessage.class)`
+
+### resourceTables
+
+Specify `resourceTables` for the API name and `analytics` for the method name to run analytics on DHIS2 as demonstrated in this route:
+
+```java
+from("direct:resourceTablesAnalytics")
+    .to("dhis2://resourceTables/analytics?skipAggregate=false&skipEvents=true&lastYears=1&username=admin&password=district&baseApiUrl=https://play.dhis2.org/2.39.1/api");
+```
+
+`dhis2://resourceTables/analytics` does not produce a new message body. The following table describes the parameters specific to `dhis2://resourceTables/analytics`:
+
+| **Parameter Name** |                              **Description**                             |
+|:------------------:|:------------------------------------------------------------------------:|
+|    skipAggregate   | Skip generation of aggregate data and completeness data.                 |
+|     skipEvents     | Skip generation of event data.                                           |
+|      lastYears     | Number of last years of data to include.                                 |
+|      interval      | Duration in milliseconds between completeness checks. Default is `30000` |
+|        async       | Whether to block until analytics is complete. Default is `false`.        |
```

Comment:
Maybe add a column specifically for default values? Also automatically explains what happens with the first 3 parameters when not supplied.